.travis.yml

```diff
@@ -280,22 +259,21 @@ before_install:
         sudo insmod $TRAVIS_INSTALL/xpmem/lib/modules/`uname -r`/kernel/xpmem/xpmem.ko
         sudo chown `whoami` /dev/xpmem
       fi
-    ## Fetch UH Tests
-    - cd $TRAVIS_SRC
-    - git clone --depth 10 https://github.com/openshmem-org/tests-uh.git tests-uh
-    # Fetch Cray Tests
-    - cd $TRAVIS_SRC
-    - git clone --depth 10 https://github.com/openshmem-org/tests-cray.git tests-cray
-    # Fetch Mellanox Tests
-    - cd $TRAVIS_SRC
-    - git clone --depth 10 https://github.com/openshmem-org/tests-mellanox.git tests-mellanox
-    ## Fetch ISx
-    - cd $TRAVIS_SRC
-    - git clone --depth 10 https://github.com/ParRes/ISx.git ISx
-    ## Fetch PRK
-    - cd $TRAVIS_SRC
-    - git clone --depth 10 https://github.com/ParRes/Kernels.git PRK
-    - echo -e "SHMEMCC=oshcc -std=c99\nSHMEMTOP=$$TRAVIS_INSTALL\n" > PRK/common/make.defs
+    - >
+      if [ -z "$SOS_DISABLE_EXTERNAL_TESTS" ]; then
```

Comment:
Should we use `-n`? We did not assign 0.

Reply:
I am confused here.. the Travis rows seem to be not downloading these anymore.. but our condition says if the string length is 0. It should say if it is non-zero (-n), right? :(

Reply:
I went with this approach because that's how we handle disabling tests elsewhere (e.g. [here](https://github.com/Sandia-OpenSHMEM/SOS/blob/v1.5.0/.travis.yml#L404), [here](https://github.com/Sandia-OpenSHMEM/SOS/blob/v1.5.0/.travis.yml#L413), and [here](https://github.com/Sandia-OpenSHMEM/SOS/blob/v1.5.0/.travis.yml#L392)).

Travis is not downloading these repos anymore because I set `SOS_DISABLE_EXTERNAL_TESTS=1` globally on line 25 of this PR.  So because $SOS_DISABLE_EXTERNAL_TESTS is nonzero, we don't clone the repos.  If we remove line 25, then the script will clone all those repos.

Reply:
Maybe it helps to keep in mind these are bash _string_ comparisons.. So it's not actually checking whether the value is 0 or non-zero, it's checking there the string is empty or non-empty (I think...).

Reply:
Ok.. thanks for pointing me to line 25. That's what I missed when I was thinking that we did not assign anything.. Good to go then